cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/common/DataprocUtils.java

```diff
@@ -22,6 +22,8 @@
 import com.google.cloud.storage.StorageBatch;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import java.util.HashMap;
```

Comment:
The current style guide has a empty line between non-java and java imports. You can refer to existing classes.

Reply:
Done